### PR TITLE
Ssb mutual (NEW pr)

### PR DESCRIPTION
**Pull request by mixmix**
_continues @dangerousbeans remote PR  https://github.com/ssbc/patchbay/pull/129_

There's currently this error when I try looking at a profile page : 
```
TypeError: this.sbot.links is not a function
    at Mutual.getAccountBalances (/home/mix/projects/SSBC/patchbay/node_modules/ssb-mutual/index.js:117:17)
    at Object.edit (/home/mix/projects/SSBC/patchbay/about/html/edit.js:88:12)
    at Object.edit (/home/mix/projects/SSBC/patchbay/node_modules/depject/apply.js:15:30)
    at Object.profilePage (/home/mix/projects/SSBC/patchbay/app/html/page/profile.js:47:40)
    at Object.page (/home/mix/projects/SSBC/patchbay/node_modules/depject/apply.js:15:30)
    at addPage (/home/mix/projects/SSBC/patchbay/app/sync/addPage.js:19:31)
    at /home/mix/projects/SSBC/patchbay/node_modules/depject/apply.js:15:30
    at api.app.async.catchLinkClick (/home/mix/projects/SSBC/patchbay/app/html/app.js:51:9)
    at HTMLDivElement.root.addEventListener (/home/mix/projects/SSBC/patchbay/app/async/catch-link-click.js:37:7)
```

### CODE REVIEW

**Comment by dangerousbeans** [comment]
`sbot plugins.uninstall ssb-backlinks` reproduces this for me

File: about/html/edit.js [hunk 1]
```diff
@@ -35,6 +38,9 @@ exports.create = function (api) {
 
   // TODO refactor this to use obs better
   function edit (id) {
+    // TODO - get this to wait till the connection is present !
+    var mutual = Mutual.init(api.sbot.obs.connection())
```
Comment (mixmix) [inline about/html/edit.js]: @dangerousbeans here was your problem - you were init'ing with `api.sbot` - this is a socket (an object which in our case had keys async and pull, not what you want).

`api.sbot.obs.connection` is an observeable which resolves to sbot when it's connected (and d_dangerously_ to null when it's not). This is not a beautiful solution and will need fixing in the future, but we can generally be sure that the sbot is connected by the time a person is browsing to a profile picture.

WORKS FOR ME!   :sob:

**Comment by mixmix** [comment]
Take this for a spin, and if it's working, let's give it a merge eh ?

**Comment by arj03** [comment]
I have tested this. Awesome work! :+1: for merging :)

File: about/html/edit.js [hunk 2]
```diff
@@ -78,6 +84,21 @@ exports.create = function (api) {
...
 
+
+    var balances_div = h('div.balances')
+
+    mutual.getAccountBalances(id, (error, balances) => {
+      if (balances == null) return ''
+
+      var balance_els = [];
+      Object.keys(balances).forEach(function(key) {
+        console.log(key)
```
Comment (arj03) [inline about/html/edit.js]: Maybe remove this?

File: about/html/edit.js (hunk 1 above)
Comment (dangerousbeans) [reply about/html/edit.js]: sweet thanks!
I had little idea of what I was doing so just found the first vaguely sbot-like thing I could find >.<

File: about/html/edit.js (hunk 2 above)
Comment (dangerousbeans) [reply about/html/edit.js]: agree - I can't push to this branch tho :C

Comment (mixmix) [reply about/html/edit.js]: hmmm, pretty sure you can @dangerousbeans ? 
https://github.com/ssbc/patchbay/settings/collaboration

Comment (mixmix) [reply about/html/edit.js]: or maybe https://github.com/ssbc/patchbay/invitations

**Pull request merged**